ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -205,7 +206,7 @@ def serialize(self, display_context: "WorkflowDisplayContext", **kwargs: Any) ->
             "id": str(node_id),
             "label": self.label,
             "type": "GENERIC",
-            **self.serialize_generic_fields(display_context),
+            **self.serialize_generic_fields(display_context, exclude=["outputs"]),
```

Comment:
this codepath should keep `outputs` serialization and consolidate with the logic above, removing line 212 below